docs/pages/experiments/slider-track-padding.tsx

```diff
@@ -0,0 +1,52 @@
+import * as React from 'react';
+import * as Slider from '@base_ui/react/Slider';
+import { Styles, TrackFill } from './slider';
+
+export default function App() {
+  return (
+    <div className="App">
+      <Slider.Root className="MySlider" defaultValue={50}>
+        <Slider.Output className="MySlider-output" />
+        <Slider.Track className="MySlider-track">
+          <TrackFill className="MySlider-track-fill" />
+          <Slider.Thumb className="MySlider-thumb one" />
+        </Slider.Track>
+      </Slider.Root>
+
+      <Slider.Root className="MySlider" defaultValue={50}>
+        <Slider.Output className="MySlider-output" />
+        <Slider.Track className="MySlider-track2">
+          <span className="MySlider-rail">
+            <TrackFill className="MySlider-track-fill" />
+          </span>
+          <Slider.Thumb className="MySlider-thumb one" />
+        </Slider.Track>
```

Comment:
@colmtuite An alternative to using a pseudo element to separately style the visible and non-visible parts of the track